projects/ngx-d3timeline/src/lib/options/options.selectors.ts

```diff
@@ -26,3 +27,16 @@ export const selectResourceAxisOptions = createSelector(
   selectOptions,
   options => options.resourceAxis
 );
+
+const getShowGridLines = (axisOptions: AxisOptions) =>
```

Comment:
maybe put this as a named function in options-utils to keep consistent with how its done elsewhere

Reply:
changed